src/config/IAppConfig.ts

```diff
@@ -66,6 +66,8 @@ export interface IServerConfig {
     api_cache_limit: number;
     show_hotspot: boolean | undefined;
     show_oncokb: boolean;
+    show_cbioportal: boolean;
```

Comment:
i think this should be `oncoprint_show_cbioportal_annot` and `oncoprint_show_cosmic_annot`